public/stylesheets/navbar.css

```diff
@@ -0,0 +1,52 @@
+.navbar-container {
+  display: flex;
```

Comment:
I was reading [Google's style guide for CSS](https://google.github.io/styleguide/htmlcssguide.html#CSS) and it mentions alphabetizing your property names.